services/system/SetCode/src/RSetCode.cpp

```diff
@@ -1,17 +1,40 @@
 #include <services/system/RSetCode.hpp>
 
 #include <psibase/dispatch.hpp>
+#include <psibase/nativeTables.hpp>
+#include <psibase/serveGraphQL.hpp>
 #include <psibase/serveSchema.hpp>
 #include <services/system/SetCode.hpp>
 
 namespace SystemService
 {
+   using namespace psibase;
+
+   struct Query
+   {
+      auto code(AccountNumber account) const
+      {
+         return kvGet<CodeRow>(CodeRow::db, codeKey(account));
```

Comment:
Needing to make two separate queries is not very GraphQLish. The result should have a member that will follow the link to the CodeByHashRow.

Reply:
The reason I didn't do this is because I didn't want to pay for pulling the entire code into memory on every query even if it wasn't requested.

But since you suggested this, I tried using a method resolver for the `rawCode` field and it's working! I had no idea our gql implementation supported this.